IGNITE-13968 Add snapshot events documentation pages (#8642)
apache/ignite: docs/_docs/events/events.adoc

| EVT_TX_RESUMED | A transaction is resumed. |The node where the transaction was started.
|===

== Snapshot Events

All snapshot events are instances of the link:{events_url}/SnapshotEvent.html[SnapshotEvent] class. The snapshot events

> always triggered at the node initiated a snapshot operation and associate with the particular snapshot operation name.

Reviewer: ```suggestion
are always triggered at the node initiated a snapshot operation. They are also associated with the particular snapshot operation name.
```

> These events allow you to be notified about the states and result of the cluster snapshot operation for further processing

Reviewer: ```suggestion
These events can notify you about the states and the result of the cluster snapshot operation, which can be used for further processing in
```

your custom procedures.

[cols="2,5,3",opts="header"]
|===

> | Event Type | Event Description | Where Event Is Fired

Reviewer: ```suggestion
| Event Type | Event Description | Where Event is Fired
```

| EVT_CLUSTER_SNAPSHOT_STARTED | A cluster snapshot operation is started. | The node where the snapshot operation was initiated.

> | EVT_CLUSTER_SNAPSHOT_FINISHED | A cluster snapshot operation is finished. The event fired when the snapshot operation finished on all the cluster nodes. |  The node where the snapshot operation was finished.

Reviewer: ```suggestion
| EVT_CLUSTER_SNAPSHOT_FINISHED | A cluster snapshot operation is finished. The event is fired when the snapshot operation is finished on all the cluster nodes. |  The node where the snapshot operation was finished.
```

> | EVT_CLUSTER_SNAPSHOT_FAILED | A cluster snapshot operation is failed. | The node where the snapshot operation was initiated. Note that message will contain the short reason of the failure from another node. You may check the logs for details.

Reviewer: ```suggestion
| EVT_CLUSTER_SNAPSHOT_FAILED | A cluster snapshot operation is failed. | The node where the snapshot operation was initiated. Note that the message will contain a failure description from another node. You may check the logs for details.
```
